packages/pn-personafisica-webapp/src/redux/newDelegation/actions.ts

```diff
@@ -16,7 +17,7 @@ export const createDelegation = createAsyncThunk<CreateDelegationResponse, NewDe
       },
       visibilityIds: data.selectTuttiEntiOrSelezionati === 'tuttiGliEnti' ? [] : [data.enteSelect],
       verificationCode: data.verificationCode,
-      dateto: data.expirationDate.toISOString().split('T')[0],
+        dateto: formatToSlicedISOString(data.expirationDate),
```

Comment:
Le modifiche sono ok.
Mi sorge un dubbio: il BE vuole solo la data senza l'orario? In caso sentirsi con Fabrizio Lana